tests/ui/rust-2024/prelude-migration/into-future-not-into-future.rs

```diff
@@ -0,0 +1,14 @@
+//@ revisions: e2021 e2024
+//@[e2021] edition: 2021
+//@[e2024] edition: 2024
+//@[e2024] compile-flags: -Zunstable-options
+//@ check-pass
+
+#![deny(rust_2024_prelude_collisions)]
+trait Meow {
+    fn poll(&self) {}
+}
+impl<T> Meow for T {}
+fn main() {
+    core::pin::pin!(()).poll();
+}
```

Comment:
This test doesn't match the file name and is identical to `tests/ui/rust-2024/prelude-migration/future-poll-not-future.rs`.  Probably that was copied as a template and then not adjusted.